testing/e2e/skr-tester/pkg/command/update.go

```diff
@@ -0,0 +1,157 @@
+package command
+
+import (
+	"errors"
+	"fmt"
+	"os/exec"
+	broker "skr-tester/pkg/broker"
+	"skr-tester/pkg/logger"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+type UpdateCommand struct {
+	cobraCmd          *cobra.Command
+	log               logger.Logger
+	instanceID        string
+	planID            string
+	updateMachineType bool
+	// TODO
+	updateOIDC bool
+}
+
+func NewUpdateCommand() *cobra.Command {
+	cmd := UpdateCommand{}
+	cobraCmd := &cobra.Command{
+		Use:     "update",
+		Aliases: []string{"u"},
+		Short:   "Update the instnace",
+		Long:    "Update the instnace",
+		Example: "skr-tester update -i instanceID --updateMachineType                            Update the instance with new machineType.",
```

Comment:
I think it's worth adding the required `planID` param to the example.

Reply:
Added.